src/services/fileServices/YmlFileServices/ConfigService.ts

```diff
@@ -0,0 +1,75 @@
+import { err, ok, Result } from "neverthrow"
+
+import ConfigParseError from "@root/errors/ConfigParseError"
+import {
+  ConfigFileName,
+  HtmlConfigFileName,
+  YmlConfigFileName,
+} from "@root/types/configYml"
+import { PathInfo } from "@root/types/pages"
+import { Brand } from "@root/types/util"
+
+// eslint-disable-next-line import/prefer-default-export
+export class ConfigService {
```

Comment:
do we have a test file for this new service?

Reply:
not at present, will add in a future PR